src/sentry/api/serializers/models/filechange.py

```diff
@@ -13,22 +13,27 @@ def get_attrs(self, item_list, user):
         commits = Commit.objects.filter(id__in=[f.commit_id for f in item_list]).select_related('author')
```

Comment:
so i think based on the [django docs](https://docs.djangoproject.com/en/1.7/ref/models/querysets/):
> Iteration. A QuerySet is iterable, and it executes its database query the first time you iterate over it.

your use of commits twice shouldn't execute two queries. But from talking to matt, it seems like it's probably just better to be explicit and coerce to a list so it's more obvious it's only one query. do you mind changing this to `commits = list(...`?